fdbserver/DDTeamCollection.actor.cpp

```diff
@@ -4955,6 +4955,7 @@ int DDTeamCollection::addBestMachineTeams(int machineTeamsToBuild) {
 				LocalityEntry process = tcMachineInfo->localityEntry;
 				forcedAttributes.push_back(process);
 				TraceEvent("ChosenMachine")
+				    .suppressFor(30.0)
```

Comment:
Maybe suppress for 1.0s? I'll let you decide.

Reply:
i have tried 5s, but it fails again https://github.com/apple/foundationdb/pull/11277#issuecomment-2032689380, so trying a larger value